### DISCUSSION TITLE
Symbols in name got rejected

### DISCUSSION THREAD

**Discussion by L1cardo** [community/community#132076] [GitHub Education]
### Hi everyone,

I am Uygur in China, and my name is unlike other Han Chinese, There is a dot "·" in my name which is "阿卜杜热伊木·阿卜杜力木", so my requests were rejected says "Variations of your name and nicknames are unacceptable."

**Comment by RachidZahrani** [comment]
Contact Support: Reach out to the platform’s support team to explain your name issue.

Alternative Formats: Try using your name without the dot: 阿卜杜热伊木阿卜杜力木.

Provide Documents: Submit official documents showing your full name with the dot.

Check Policies: Review the platform’s name format rules.

Explain Context: If possible, explain the cultural significance of the dot in your name.
